### PR TITLE
[scala/en] Added REPL basics to help people experiment as they learn.

### PR DESCRIPTION
**Pull request by doub1ejack**

### CODE REVIEW

File: scala.html.markdown [hunk 1]
```diff
@@ -12,22 +12,62 @@ Scala - the scalable language
 
 ```scala
 
+/////////////////////////////////////////////////
+// 0. Basics
```
Comment (geoffliu) [inline scala.html.markdown]: Hmm, maybe call this section "Setting up"? We can also combine the two chunks of comments into one.

Comment (geoffliu) [inline scala.html.markdown]: @doub1ejack ping?

**Comment by vendethiel** [comment]
@doub1ejack ping #2?:)

**Comment by geoffliu** [comment]
Eh I'll just merge this, it's a good edit despite my nitpicking.

**Pull request merged**